modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/AbstractPlannerTest.java

```diff
@@ -570,6 +570,10 @@ protected <T extends RelNode> void assertPlan(
                 return;
             }
         } catch (Throwable th) {
+            if (lastErrorMsg == null) {
```

Comment:
is it last or first?

Reply:
well, it's a bit messy... This variable is just a field of an abstract class. Every test in test class extending `AbstractPlannerTest` will overwrite this variable if validation has failed. So, this is the last error message for test class, but this is a first and only error message for test.